app/assets/javascripts/models/beta/story.js

```diff
@@ -63,6 +63,9 @@ export const isRelease = (storyType) => storyType === 'release';
 
 export const types = ['feature', 'bug', 'release', 'chore'];
 
+export const states = ['unscheduled', 'unstarted', 'started', 'finished', 'delivered',
```

Comment:
I would break this line differently, like 3 per line (with the last line having only 2) or one per line, what do you think?